admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -158,29 +159,29 @@ object DelegateApiHandler {
     def writeBuf[T](t: T): Buf = Buf.ByteArray.Owned(mapper.writeValueAsBytes(t))
   }
 
+  def getDelegateRsp(dtab: String, path: String, interpreter: NameInterpreter): Future[Response] =
+    (dtab, path) match {
+      case (DtabStr(d), PathStr(p)) =>
+        Delegator(d, p, interpreter).values.toFuture().flatMap(Future.const).map { tree =>
```

Comment:
Fair point. I've noted this in #326. We may or may not bundle all that work together, but at least it's documented either way.